### PR TITLE
Step-10B: Style Card List

### PR DESCRIPTION
**Pull request by sta-wtag**
https://user-images.githubusercontent.com/122765730/221525210-77a5377e-950c-46e6-b5f5-f1da4176c9ae.mov

### Step-10B

Add style to main UI card list.

### Acceptance Criteria

- [x] Add a grid style with 3 columns and 3 row in desktop and 1 column in mobile.

### CODE REVIEW

File: components/splashScreen.vue [hunk 1]
```diff
@@ -0,0 +1,48 @@
+<template>
+  <div class="main-div flex-box">
+    <div class="align-self-center content relative-position">
+      <div class="icon-anim"></div>
+      <div class="flex-box">
+        <SplashIcon class="align-self-center" />
+        <div class="splash-text margin-left-28">TODOS</div>
```
Comment (sai-wtag) [inline components/splashScreen.vue]: we can use i18n for ''TODOS"

File: jest.config.js [hunk 2]
```diff
@@ -1,17 +1,11 @@
...
-    'ts', // if using TypeScript
-    'js',
-    'vue',
-    'json',
-  ],
+  // moduleNameMapper: {
+  //   '^@/(.*)$': '<rootDir>/$1',
+  //   '^~/(.*)$': '<rootDir>/$1',
+  //   '^vue$': 'vue/dist/vue.common.js',
+  // },
```
Comment (sai-wtag) [inline jest.config.js]: Please remove, if not needed

File: tests/unit/TaskCard.spec.js [hunk 3]
```diff
@@ -1,22 +1,68 @@
+import { describe, expect, it } from '@jest/globals';
 import { mount } from '@vue/test-utils';
-// import Nuxti18n from '@nuxtjs/i18n';
```
Comment (sai-wtag) [inline tests/unit/TaskCard.spec.js]: ditto

File: tests/unit/TaskCard.spec.js [hunk 4]
```diff
@@ -1,22 +1,68 @@
...
-import TaskCard from '@/components/TaskCard.vue';
+import Vuex from 'vuex';
+import NuxtI18n from 'vue-i18n';
+import localVue from '../utils/vueInstanceFactory';
+import i18Mock from '../utils/i18Mock';
+import { todos } from '../utils/storeHelper';
+// import lang from '../../store/lang';
```
Comment (sai-wtag) [inline tests/unit/TaskCard.spec.js]: ditto

File: tests/unit/TaskCard.spec.js [hunk 5]
```diff
@@ -1,22 +1,68 @@
...
+import { helper } from '../../helpers/helper';
+import TaskCard from '../../components/TaskCard.vue';
 
-async function wrapperFactory(newState = {}) {
-  const mounted = await mount(TaskCard, {
-    // Nuxti18n,
```
Comment (sai-wtag) [inline tests/unit/TaskCard.spec.js]: ditto

File: tests/utils/storeHelper.js [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+//   getters: {
+//     currentAppUser() {},
+//   },
+
+//   actions: {
+//     fetchUserById: jest.fn(),
+//     updateUser: jest.fn(),
+//     addSchedules: jest.fn(),
+//   },
+// };
```
Comment (sai-wtag) [inline tests/utils/storeHelper.js]: ditto

**Review by sai-wtag — APPROVED** [review]
Some i18n warnings are showing in the console.
Other than that, LGTM. Thanks

Screenshot 2023-02-20 at 11 19 11 AM

File: jest.config.js (hunk 2 above)
Comment (sta-wtag) [reply jest.config.js]: Okay. Thank you

File: components/splashScreen.vue (hunk 1 above)
Comment (sta-wtag) [reply components/splashScreen.vue]: Should we translate splash screen texts as it is similar to a brand logo?

File: tests/unit/TaskCard.spec.js (hunk 3 above)
Comment (sta-wtag) [reply tests/unit/TaskCard.spec.js]: Okay. Thank you

File: tests/unit/TaskCard.spec.js (hunk 4 above)
Comment (sta-wtag) [reply tests/unit/TaskCard.spec.js]: Okay. Thank you

File: tests/unit/TaskCard.spec.js (hunk 5 above)
Comment (sta-wtag) [reply tests/unit/TaskCard.spec.js]: Okay. Thank you

File: tests/utils/storeHelper.js (hunk 6 above)
Comment (sta-wtag) [reply tests/utils/storeHelper.js]: Okay. Thank you

**Review by hhu-wtag — CHANGES_REQUESTED** [review]
Some design related issues found.

Screen Shot 2023-02-22 at 10 06 38 AM

Screen Shot 2023-02-22 at 10 10 19 AM

Screen Shot 2023-02-22 at 10 11 47 AM

Screen Shot 2023-02-22 at 10 13 06 AM

Screen Shot 2023-02-22 at 10 15 01 AM

**Review by bia-wtag — CHANGES_REQUESTED** [review]
Tested locally. Please have a look at the following issues, 

- Issues from https://github.com/sta-wtag/TodoApp/pull/12#pullrequestreview-1314609472 still remain.
- Changing the filter always shows the same placeholder, could be updated.
- Language selector isn't at the right place for mobile
Screenshot 2023-02-26 at 10 50 35 AM

- Alignment broken for smaller screens
Screenshot 2023-02-26 at 10 51 04 AM
- The position for the toaster on mobile can be updated.
https://user-images.githubusercontent.com/85149436/221393058-d2bd181a-6ec2-4cc0-85ad-41ae77fce1e5.mov

**Review by hhu-wtag — CHANGES_REQUESTED** [review]
Screen Shot 2023-03-02 at 12 06 49 PM

image

**Review by hhu-wtag — APPROVED** [review]
image

**Review by bia-wtag — APPROVED** [review]

**Pull request merged**